clang-tools-extra/clangd/ModulesBuilder.cpp

```diff
@@ -318,6 +309,10 @@ bool StandalonePrerequisiteModules::canReuse(
   Clang.getHeaderSearchOpts().ForceCheckCXX20ModulesInputFiles = true;
   Clang.getHeaderSearchOpts().ValidateASTInputFilesContent = true;
 
+  // Following the practice of clang's driver to suppres the checking for ODR
```

Comment:
can you instead point to https://clang.llvm.org/docs/StandardCPlusPlusModules.html#object-definition-consistency ?

Reply:
Done